bundles/asdi/asdi-projection-change/component/card.js

```diff
@@ -0,0 +1,65 @@
+Oskari.clazz.define('Oskari.projection.change.component.card',
+/**
+ * @class 
+ * @param { Object } view - contains info about the current view passed down to card
+ * @param function() callback - callback function to call when element is clicked
+ */
+function (view, callback) {
+    this.card = _.template('<div class="projection-card" data-srs="${srs}">'+
+                        '<div class="card-image ${imgCls}"></div> '+
+                        '<div class="info-row">'+
+                            '<p class="card-header"> ${projectionName} </p>'+
+                            '<div class="projection-info icon-info"></div>'+
+                            '<div class="projection-warning" title="${tooltip}"></div>'+
+                        '</div>'+
+                        '</div>');
+    this.element = null;
+    this.loc = Oskari.getLocalization('projection-change');
+    this.callback = callback;
+    this.view = view;
+    this.infoView = Oskari.clazz.create('Oskari.projection.change.view.ProjectionInformation', view);
+    this.errorListing = Oskari.clazz.create('Oskari.projection.change.view.ErrorListing');
+    if ( !this.getElement() )
+    {
+        this.create(view);
+    }
+}, {
+    getElement: function () {
+        return this.element;
+    },
+    setElement: function (el) {
+        this.element = el;
+    },
+    createClassSelector: function ( crs ) {
+        return crs.replace(':', '');
```

Comment:
Is this required? The styling that you use is for selector: [data-srs="EPSG:3576"] .card-image so having the srs as class isn't used anywhere?